db/migrate/20160915144154_add_color_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddColorToTags < ActiveRecord::Migration
+  def change
+    add_column :tags, :color, :string, default: "#f50"
```

Comment:
what's the benefit of having the colour inside the model itself as an attribute? Why not just style it w/ css?